packages/plugins/src/SidebarPlugin.ts

```diff
@@ -172,14 +152,18 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
-      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
+      //Map into Sidebar Groups and sort children according to priority
       function sortSidebarGroups(sidebarData) {
         const sortedGroupedPages = sidebarData.map(page => {
-          if (page.childNodes) {
-            const sortedPages = page.childNodes.sort((a, b) => b.priority - a.priority);
+          if (page.childNodes.length > 1) {
+            const sortedPages = page.childNodes.sort(
+              (pageA, pageB) =>
+                (pageB.priority ? pageB.priority : -1) - (pageA.priority ? pageA.priority : -1)
```

Comment:
the original sort would prioritise the index pages of the current directory/group
 this now just depends on priority
 
 Are the sorts equivalent, probably worth adding a description to this PR, so we can see what we changing.
 
 See @tomhazledine latest pr, which has a great overview of the search change